guides/common/modules/proc_disabling-puppet.adoc

```diff
@@ -0,0 +1,24 @@
+[id="Disabling_Puppet_Integration_{context}"]
+= Disabling Puppet Integration with {Project}
+
+To discontinue using Puppet in your {Project}, follow this procedure.
+
+.Prerequisites
+* You upgraded {Project} from the previous version with Puppet server enabled.
```

Comment:
Do you need to have upgraded from a previous version? Another path to get here is to install Satellite 7.0 without Puppet and then follow the enabling procedure. I'm not sure how to best phrase it but essentially it can be generalized as "Puppet is enabled on {Project}".